### ISSUE TITLE
<pshome>/<culture>/default.help.txt is missing

### ISSUE THREAD

**Issue by TravisEz13** [PowerShell/PowerShell#1731]
# Expected

<pshome>/<culture>/default.help.txt is missing
# Actual

```
PS C:\Program Files\PowerShell\6.0.0.7> dir .\en-US\


    Directory: C:\Program Files\PowerShell\6.0.0.7\en-US


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----         8/5/2016   2:05 PM        2974257 System.Management.Automation.dll-help.xml
```

``` powershell
> $PSVersionTable

Name                           Value
----                           -----
PSEdition                      Core
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
CLRVersion
PSRemotingProtocolVersion      2.3
BuildVersion                   3.0.0.0
WSManStackVersion              3.0
GitCommitId                    v6.0.0-alpha.7-378-g86c5d3e4c88fe1489dcb657132e64dd8535da6a1
SerializationVersion           1.1.0.1
PSVersion                      6.0.0-alpha

```

**Comment by Francisco-Gamino** [comment]
@joeyaiello: I am assigning this to you as a reminder to add default.txt to the Microsoft.PowerShell.Core help content cab and zip packages.

**Comment by Francisco-Gamino** [comment]
This issue was fixed when I enable get-help on Linux. Please see https://github.com/PowerShell/PowerShell/pull/1695. Thanks.

**Comment by Francisco-Gamino** [comment]
This issue is fixed on Linux, but not on Windows, so activating issue.

**Comment by Francisco-Gamino** [comment]
This issue has been fixed by #3032.
